kairos-cli/src/lib.rs

```diff
@@ -1,4 +1,44 @@
+pub mod client;
 pub mod commands;
 pub mod common;
 pub mod error;
 pub mod utils;
+
+use crate::error::CliError;
+
+use clap::{Parser, Subcommand};
+use reqwest::Url;
+
+#[derive(Parser)]
+#[command(name = "Kairos Client", about = "CLI for interacting with Kairos")]
+pub struct Cli {
+    #[command(subcommand)]
+    pub command: Command,
+    #[arg(long, value_name = "URL")]
+    pub kairos_server_address: Option<Url>,
+}
+
+#[derive(Subcommand)]
+pub enum Command {
+    #[command(about = "Deposits funds into your account")]
+    Deposit(commands::deposit::Args),
+    #[command(about = "Transfers funds to another account")]
+    Transfer(commands::transfer::Args),
+    #[command(about = "Withdraws funds from your account")]
+    Withdraw(commands::withdraw::Args),
+}
+
+pub fn run(
+    Cli {
+        command,
+        kairos_server_address,
+    }: Cli,
+) -> Result<String, CliError> {
+    let kairos_server_address =
+        kairos_server_address.unwrap_or(Url::parse("http://0.0.0.0:9999").unwrap());
```

Comment:
`http://0.0.0.0:9999`?

Reply:
I thought I'll make the `kairos-server-address` an optional argument, and if users don't specify it, we default to some address. I chose a random local one. I'm open to making this a mandatory argument and then users would always need to specify a URL

Reply:
It should be mandatory argument, but with default value, so user can skip it - see https://github.com/cspr-rad/kairos/pull/91/files#r1590933505.

Reply:
Code removed in https://github.com/cspr-rad/kairos/pull/91/commits/0c980efa500346d61c26d90b31d252e9e8403b5e.